patches.python/0002-libvirtaio-cache-the-list-of-callbacks-when-calling.patch

```diff
@@ -0,0 +1,43 @@
+From 444a790e638acef38fecf18387c8b91732d4eede Mon Sep 17 00:00:00 2001
+From: Wojtek Porczyk <woju@invisiblethingslab.com>
+Date: Thu, 31 Aug 2017 21:20:40 +0200
+Subject: [PATCH 2/6] libvirtaio: cache the list of callbacks when calling
+Organization: Invisible Things Lab
+Cc: Wojtek Porczyk <woju@invisiblethingslab.com>
+
+When the callback causes something that results in changes wrt
+registered handles, python aborts iteration.
+
+Relevant error message:
+
+    Exception in callback None()
+    handle: <Handle cancelled>
+    Traceback (most recent call last):
+      File "/usr/lib64/python3.5/asyncio/events.py", line 126, in _run
+        self._callback(*self._args)
+      File "/usr/lib64/python3.5/site-packages/libvirtaio.py", line 99, in _handle
+        for callback in self.callbacks.values():
+    RuntimeError: dictionary changed size during iteration
+
+QubesOS/qubes-issues#2805
```

Comment:
Would you mind putting this reference into commit message (adding this patch) too? Preferably with `Fixes` keyword.